<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog1</class>
 <widget class="QDialog" name="Dialog1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose mode</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Against AI</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>95</width>
     <height>103</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_AI_easy">
      <property name="text">
       <string>easy</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_AI_medium">
      <property name="text">
       <string>medium</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_AI_hard">
      <property name="text">
       <string>hard</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Two players</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_two_player">
      <property name="text">
       <string>click</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_history">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>view history</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
